docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,99 @@
+
+# CHW & CW - Rule 22-42  
+
+**Rule ID:** 22-42   
+**Rule Description:** The sets of performance curves specified in Table J-2 should be used to represent part-load performance of chillers in the baseline building design.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.2.2.1 Baseline 
+
+**Mandatory Rule:** True  
+**Evaluation Context:** Each Chiller  
+**Table Lookup:**
+J-6
+**Function Call:**  
+
+## Applicability Check:  
+- look at each chiller - only water-cooled chillers are applicable (ie chillers with a condensing_loop): `for chiller in B_RMD.chillers:`
+    - if the chiller has a condensing_loop, and the chiller energy source is electricity, continue to rule logic: `if chiller.consensing_loop != NULL and chiller.energy_source_type == "ELECTRICITY": CONTINUE TO RULE LOGIC`
+
+        ## Rule Logic:
+        - figure out the set of performance curves needed for the chiller - this should be V, X, Y, Z, AA or AB.  Initialize a variable curve_set and set it to "NONE": `curve_set = "NONE"`
```

Comment:
@weilixu I imagine that the actual implementation would use table lookups here. Is it clear enough from this RDS without using table lookups?